app/components/dashboard/replication_status_component.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="card border-0 text-start">
+  <div class="card-body" />
+    <div class="card-title fs-4 fw-semibold d-flex justify-content-between">
+      Replication Zips
+      <span class="badge rounded-pill <%= replication_and_zip_parts_ok? ? 'bg-success' : 'bg-danger' %>">
+        <%= replication_and_zip_parts_ok? ? 'OK' : 'Error' %>
+      </span>
+    </div>
+    <ul class="list-group border border-2 border-secondary">
+      <li class="list-group-item d-flex justify-content-between align-items-center">
+        <h6 class="d-flex justify-content-start">ZipPart Statuses</h6>
+        <span class="badge rounded-pill <%= zip_parts_ok? ? 'bg-success' : 'bg-danger' %>">
+          <%= zip_parts_ok? ? 'OK' : 'Error' %>
+        </span>
+      </li>
+      <% endpoint_data.each do |endpoint_name, endpoint_info| %>
+        <% replication_count = endpoint_info[:replication_count] %>
```

Comment:
Smell that this should be its own component.

Reply:
Good point!  I will ticket as I believe it will clash with PR #2086

Reply:
ticketed as #2089